tests/e2e/testfiles/authorization-templates/storage_csm_authorization_v1_proxy_server_no_cert.yaml

```diff
@@ -52,9 +52,8 @@ spec:
           proxyServerIngress:
             - ingressClassName: nginx
               # additional host rules for the proxy-server ingress
-              hosts: []
-              # - [application name]-ingress-nginx-controller.[namespace].svc.cluster.local
-
+              hosts:
```

Comment:
does this test pass the proxy-ns auth e2e test case?

Reply:
https://github.com/dell/csm-operator/blob/4a93164861034440d5646fb915d026c41eb0f747/tests/e2e/testfiles/scenarios.yaml#L1159 If this is the test scenario you are referring to... 

Yes, this test scenario still passes. @JacobGros and I are working on ensuring all --pflex E2E scenarios pass and we won't merge the PR until there is a log proving as such.